demo/chbench/mzcompose.yml

```diff
@@ -31,17 +31,19 @@ services:
     ports:
      - *materialized
     init: true
-    command: -w ${MZ_THREADS:-1}
-    environment:
-      # you can for example add `pgwire=trace` or change `info` to `debug` to get more verbose logs
-      - MZ_LOG=pgwire=debug,info
+    command:
```

Comment:
You can use the block chomper (`>-`) to avoid the need for extra strings.

```
command: >-
    --workers=${MZ_WORKERS:-1}
    --dataflow-eager-merge=1000
    --dataflow-progress-mode=${MZ_DATAFLOW_PROGRESS_MODE:-eager}
```

Reply:
Unfortunately that doesn't allow for inline comments.

Reply:
That's a shame.